### PR TITLE
final-milestone: Metric App

### PR DESCRIPTION
**Pull request by Simpleshaikh1**
When the page loads, the webapp shows the list of categories that could be filtered out by some parameter.
Along with the category name, some numeric values per category that come from the API are displayed.
When the user clicks (or taps) on a category item, the application navigates to the details page.
In the details page, the webapp retrieves data from the API to show detailed data in the category.
When the user clicks on the "Back" button (<), the user navigates to the home page.
Created unit tests for pure functions in Redux code.
Created integration tests for my application using the React Testing Library.
I mocked the access to the API, so that tests don't send actual requests.
I mocked the connection to the Redux Store.

### CODE REVIEW

File: src/redux/coinlore/CoinSlice.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ export const getCoins = createAsyncThunk('crypto/fetchCrypto', async (_, api) =>
     const res = await axios.get(getCoinUrl);
     return res.data;
   } catch (error) {
-    return api.rejectWithValue('Api failed to fetch');
+    return api.rejectWithValue('Api failed to fetch data');
```
Comment (Munsa1) [inline src/redux/coinlore/CoinSlice.js:17]: - [ ] As requested by the previous reviewer in this [comment](https://github.com/Simpleshaikh1/Metric-App/pull/5#discussion_r1320547407) on the first [Pr](https://github.com/Simpleshaikh1/Metric-App/pull/5), kindly ensure your design follows the given design

Your design
image

Given design
image

- Kindly use the given font family for all the text in your app
image

- The layout should also be followed, make sure there is no space around the cards, they should span the whole view. Also text should be aligned to the bottom left as in the given design

Your cards
image

Given cards
image

File: src/redux/coinlore/CoinSlice.js [hunk 2]
```diff
@@ -14,7 +14,7 @@ export const getCoins = createAsyncThunk('crypto/fetchCrypto', async (_, api) =>
     const res = await axios.get(getCoinUrl);
```
Comment (Munsa1) [inline src/redux/coinlore/CoinSlice.js:14]: - [ ] You have some failing test in `App.js`, kindly fix the failimg test
image

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @Simpleshaikh1, 
required changes

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


## Suggested changes
_Check the comments under the review._
## :warning: Please do not open another pull request after this review, kindly implement the changes here :warning: 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by tugsem** [comment]
## STATUS: APPROVED ✔️ 
Hi @Simpleshaikh1 , 

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 

Alt Text
Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**